frontend/app/src/composables/api/assets/info.ts

```diff
@@ -24,18 +33,23 @@ export function useAssetInfoApi(): UseAssetInfoApiReturn {
     return AssetMap.parse(handleResponse(response));
   };
 
-  const assetSearch = async (
-    keyword: string,
-    limit = 25,
-    searchNfts = false,
-    signal?: AbortSignal,
-  ): Promise<AssetsWithId> => {
+  const assetSearch = async (params: AssetSearchParams): Promise<AssetsWithId> => {
+    const {
+      value,
+      evmChain,
+      limit,
+      searchNfts,
+      signal,
+      address,
+    } = params;
     const response = await api.instance.post<ActionResult<AssetsWithId>>(
       '/assets/search/levenshtein',
       snakeCaseTransformer({
-        value: keyword,
-        limit,
+        value,
+        evmChain,
+        limit: limit || 25,
```

Comment:
just small fyi this could become a `limit = 25` on the destructuring of params above